x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/routes/init.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { buildSiemResponse } from '@kbn/lists-plugin/server/routes/utils';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { ENTITY_STORE_INIT_URL } from '../../../../../common/constants';
+import type { SecuritySolutionPluginRouter } from '../../../../types';
+export const entityStoreInitRoute = (router: SecuritySolutionPluginRouter) => {
+  router.versioned
+    .post({
+      access: 'internal',
+      path: ENTITY_STORE_INIT_URL,
+      options: {
+        tags: ['access:securitySolution'], // TODO entity store access `access:${APP_ID}-entity-analytics`
+      },
+    })
+    .addVersion(
+      { version: '1', validate: {} },
+      // TODO Implement entity store privileges like `withRiskEnginePrivilegeCheck` in risk_engine_privileges.ts
```

Comment:
I am not sure this TODO is needed just yet. We needed `withRiskEnginePrivilegeCheck` because we wanted all risk engine routes to require the same privileges even though they didn't technically need them.

For the entity store we will probably be able to have a simpler approach more in line with asset criticality APIs